src/Personal/reducer.js

```diff
@@ -0,0 +1,67 @@
+import { actionType } from './actionCreator';
+
+const initialBackup = {
+  response: 'Click button to backup data',
+};
+
+const initialUser = {
+  username: '',
+  email: '',
+  accessToken: '0xa143981f3ec758296a1575146eab03ef047b7e40',
+};
+
+const backupState = (state = initialBackup, action) => {
+  switch (action.type) {
+    case actionType.BACKUP_START:
+      return {
+        ...state,
+        response: 'Backing up',
+      };
+    case actionType.BACKUP_DATA:
+      return {
+        ...state,
+        response: 'Back up successfully!',
+      };
+    case actionType.BACKUP_FAILED:
+      return {
+        ...state,
+        response: 'Ops, something went wrong',
+      };
+    case actionType.BACKUP_INITIAL:
```

Comment:
We don't really need this action. You can decide on the rendered button based on the last backup date.

Reply:
Done